### PR TITLE
Add: Unit Test source for multiple conn

### PR DESCRIPTION
**Pull request by MilanBariya**
### Describe your changes :

I worked on adding a Unit Test source for multiple connections

#
### Type of change :

- [x] Improvement

### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

**Comment by ayush-shah** [comment]
1 bug from sonar cloud will be handled in #4904

**Comment by ayush-shah** [comment]
> 1 bug from sonar cloud will be handled in #4904

My bad, it was already fixed

File: ingestion/tests/unit/test_source_connection.py [hunk 1]
```diff
@@ -209,3 +241,365 @@ def test_pinotdb_url(self):
...
+    def test_clickhouse_url(self):
+        # connection arguments without db
+        expected_url = "clickhouse+http://username:@localhost:8123"
+        clickhouse_conn_obj = ClickhouseConnection(
+            username="username",
+            hostPort="localhost:8123",
+            scheme=ClickhouseScheme.clickhouse_http,
+            database=None,
+        )
+        assert expected_url == get_connection_url(clickhouse_conn_obj)
```
Comment (ayush-shah) [inline ingestion/tests/unit/test_source_connection.py:275]: Does it work without password?
are we able to establish connection?

Comment (MilanBariya) [reply ingestion/tests/unit/test_source_connection.py:275]: @ayush-shah yes it's work without password.

**Review by ayush-shah — APPROVED** [review]

**Comment by ayush-shah** [comment]
Hey @pmbrull - anything from your side?

**Pull request merged**